src/core/linked-list.ts

```diff
@@ -36,11 +36,14 @@ export class LinkedList<T> {
 	 * @param value
 	 * @returns The added node.
 	 */
-	addAfter(node: LinkedListHeadNode<T> | LinkedListMiddleNode<T>, value: T): LinkedListMiddleNode<T> {
+	addAfter (
+		node: LinkedListHeadNode<T> | LinkedListMiddleNode<T>,
+		value: T,
+	): LinkedListMiddleNode<T> {
 		// assumes that node != list.tail && values.length >= 0
 		const next = node.next;
 
-		const newNode = { value, prev: node, next };
+		const newNode = { value , prev : node , next  };
```

Comment:
This also seems like a bug with prettier-plugin-space-before-function-paren. I'll take a look at that.

Reply:
Fixed in [`v0.0.6`](https://www.npmjs.com/package/prettier-plugin-space-before-function-paren) of the plugin.

Reply:
Updated and it does fix, thanks